Modules/System/SharePoint/SPClientImpl.Codeunit.al

```diff
@@ -0,0 +1,365 @@
+codeunit 9101 "SP Client Impl."
+{
+    //Access = Internal;
+
+    var
+        UriBuilder: Codeunit "SP Uri Builder";
```

Comment:
I do really appreciate this PR, but mabye the Variables should be changed to Include the SP prefix, since there exiss another Codeunit "Uri Builder" in the System Application. This could be very misleading.

Reply:
I think this is fair point! Also the name of the file would be changed - it also should reflect SPUriBuilder